### ISSUE TITLE
status

### ISSUE THREAD

**Issue by dennislabela** [authindicators/rfc-brand-indicators-for-message-identification#17]
what's the status of this project? I see the first draft is expired, and resubmitted, but not a lot has been changed.

https://www.ietf.org/rfcdiff?url1=draft-brotman-ietf-bimi-guidance-00&url2=draft-brotman-ietf-bimi-guidance-01&difftype=--html

**Comment by abrotman** [comment]
No, not much has changed with the drafts. We've been working on trials at some providers.  I believe the goal is to go back the IETF based on findings/changes from trials.

http://www.bimigroup.org/

Is another resource that can be used.  There's a contact form on that page as well.
